charts/wildfly/Chart.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 name: wildfly
 description: Build and Deploy WildFly applications on OpenShift
 type: "application"
-version: 1.4.0
+version: 1.4.1
 
 # This is the version number of the application being deployed.
 # This maps to the WildFly S2I Images tag that is used for S2I build.
-appVersion: "24.0"
+appVersion: "24.0.1"
```

Comment:
@yersan It was not necessary to change the `appVersion` as the `24.0.1` image tag is also linked to the `24.0` tag in https://quay.io/repository/wildfly/wildfly-centos7?tag=24.0&tab=tags

Reply:
Alright Jeff, thanks for the information. Then `major.minor` will always include the latest patches.